### ISSUE TITLE
Универсальный наряд

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2119]
Originally reported on Google Code with ID 2119

```
Универсальный наряд. Пользователь должен иметь возможность создавать свои формулы для
расчета надбавок.

```

Reported by `roberto1813` on 2010-09-21 16:20:29

**Comment by gsbelarus** [comment]
Reported by `roberto1813` on 2010-09-21 16:29:30
- **Labels added**: WageFC3
- **Labels removed**: WageFC

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-09-21 16:29:49
- **Labels added**: Wage, FC2

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-09-21 16:30:55
- **Labels removed**: WageFC3

**Comment by gsbelarus** [comment]
Reported by `alexandra.gsoftware` on 2011-07-25 18:39:56
- **Status changed**: `Duplicate`
- **Merged into**: #1886
